pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "dnspython";
```

Comment:
this is a duplicate of the attribute `dns`

Reply:
History. It was simply called `dns` because it was in an expression containing only Python packages. So why duplicate the fact that it its a Python package?

We could introduce an alias and then with 18.09 remove the `dns` attribute.

Reply:
@sebix that's also the current guideline in `python-packages.nix`